jf_agent/git/gitlab_client.py

```diff
@@ -31,13 +31,14 @@ def log_and_print_request_error(e, action='making request', log_as_exception=Fal
 
 
 class GitLabClient:
-    def __init__(self, server_url, private_token, verify, per_page_override, session):
+    def __init__(self, server_url, private_token, verify, per_page_override, session, version):
         kwargs = {'private_token': private_token, 'session': session}
         if per_page_override is not None:
             kwargs['per_page'] = per_page_override
         if not verify:
             kwargs['ssl_verify'] = False
         self.client = gitlab.Gitlab(server_url, **kwargs)
+        self.version = '4'
```

Comment:
To avoid future confusion I'd rename this to `self.api_version` -- it's the version of the GitLab API that's 3 vs. 4, not the version of GitLab itself

Reply:
done